config.js

```diff
@@ -23,6 +23,11 @@ mongo_hacker_config = {
   force_color:     false,           // force color highlighting for Windows users
   column_separator:  '→',           // separator used when printing padded/aligned columns
   value_separator:   '/',           // separator used when merging padded/aligned values
+  better_dbref: {
```

Comment:
i think this might be better as:

```
  ...
  dbref: {
    extended_info: bool,
    plain: bool,
    db_if_differs: bool
   }
  ...
```

comments are great though